packages/client-web-kit/src/components/entities/policy/APolicyBasicForm.vue

```
@@ -10,8 +10,8 @@ import useVuelidate from '@vuelidate/core';
 import { maxLength, minLength, required } from '@vuelidate/validators';
 import type { Policy } from '@authup/core-kit';
 import { IVuelidate } from '@ilingo/vuelidate';
-import type { FormSelectOption } from '@vuecs/form-controls';
-import { VCFormGroup, VCFormInput, VCFormInputCheckbox } from '@vuecs/form-controls';
+import type { FormOption } from '@vuecs/forms';
+import { VCFormGroup, VCFormInput, VCFormCheckbox } from '@vuecs/forms';
```

ESLint

[error] Member 'VCFormCheckbox' of the import declaration should be sorted alphabetically.

(sort-imports)